action.yml

```diff
@@ -2,7 +2,7 @@
 
 name: Setup MATLAB
 description: >-
-  Set up MATLAB on a Linux-based GitHub-hosted runner
+  Set up a specific version of MATLAB and MathWorks products
```

Comment:
Is it possible to remove "and MathWorks products"?

Reply:
Updated